app/src/test/java/com/example/trainingintive/dog_images_feature/domain/usecase/GetDogImageUrlUseCaseTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.trainingintive.dog_images_feature.domain.usecase
+
+import com.example.trainingintive.dog_images_feature.domain.model.DogImageUrl
+import com.example.trainingintive.dog_images_feature.domain.repository.DogImageRepository
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import io.reactivex.rxjava3.core.Single
+import org.junit.Test
+
+class GetDogImageUrlUseCaseTest {
+
+    val dogImageUrl =
+        DogImageUrl("https:\\/\\/images.dog.ceo\\/breeds\\/bulldog-french\\/n02108915_9666.jpg")
+    val repository: DogImageRepository = mockk() {
+        every { getDogImageUrl() } returns Single.fromCallable { dogImageUrl }
```

Comment:
Use `Single.just(...)`

Reply:
Also, such mocking should be in test function because maybe in other tests (later) you will need to test errors, and having mock always returning positive value for all tests can make a fuss